README.md

```diff
@@ -90,42 +88,64 @@ The [**State-fold Server**](./offchain/state-server/README.md) is the component
 
 The [**Dispatcher**](./offchain/dispatcher/README.md) is the component that messages *Inputs* via the **Broker** to be processed elsewhere and submits *Claims* to the blockchain.
 
+The [**Authority Claimer**](./offchain/authority-claimer/README.md) is the component that submits *Claims* to the blockchain at the end of every epoch.
+
 The [**Advance Runner**](./offchain/advance-runner/README.md) is the one responsible for relaying *Inputs* to the **Server Manager** to be processed by the underlying DApp running in the embedded **Cartesi Machine**.
 The **Advance Runner** obtains the resulting **Outputs** and **Claims** from the **Server Manager** and adds them to the **Broker**.
 
 The [**Indexer**](./offchain/indexer/README.md) consumes all *Inputs* and *Outputs* transferred by the **Broker** and stores them in a **PostgreSQL** database for later querying via a **GraphQL Server** instance.
 
 The [**Inspect Server**](./offchain/inspect-server/README.md) is responsible for the processing of *Inspect* requests via HTTP.
 
-## Workflows
+## Framework features
 
-The Cartesi Node may act as a *User* (aka *Reader*) or *Validator* Node, hence performing different roles, which may be represented by different workflows.
+The Cartesi Node contains 3 main features, the combination of which defines the node capabilities
 
-### Reader Flow
+### Reader Feature
 
-In the Reader Flow, any *Input* read from the blockchain by the **State-fold Server** is received by the **Dispatcher** and relayed to the **Broker** through an input-specific stream.
+With the Reader Feature, any *Input* read from the blockchain by the **State-fold Server** is received by the **Dispatcher** and relayed to the **Broker** through an input-specific stream.
```

Comment:
This paragraph seems strange because the text describes details of the implementation instead of the feature itself.
We first need an explanatory paragraph for each feature. For instance, in the processing inputs feature, we could have something like:

> The Node is capable of processing rollup inputs stored in the blockchain. To do that, the node gets the inputs from the blockchain, sends them to the Cartesi machine, and stores the result in the node database. Once the input result is in the database, the node client can query the outputs in the GraphQL API.